static/explorer.scss

```diff
@@ -91,6 +91,11 @@ body {
     width: 99%;
 }
 
+.change-device {
+    min-width: 14em;
```

Comment:
From my experimentation, and assuming we don't add too many other tool icons on the view, I think we should make this larger, maybe 18 or 20?